src/libYARP_dev/src/devices/ControlBoardWrapper/RPCMessagesParser.cpp

```diff
@@ -234,7 +235,8 @@ void RPCMessagesParser::handleControlModeMsg(const yarp::os::Bottle& cmd,
                     {
                         modes[i] = modeList->get(i).asVocab();
                     }
-                    *ok = rpc_iCtrlMode2->setControlModes(modes);
+                    if(rpc_iCtrlMode2)
+                        *ok = rpc_iCtrlMode2->setControlModes(modes);
```

Comment:
See previous comment.

Reply:
Fixed in https://github.com/robotology/yarp/commit/6b79a0c894708c3ef19992997f1ad1ba8968f39e